charts/prometheus-logstash-exporter/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+description: Logstash stats exporter for Prometheus
+name: prometheus-logstash-exporter
+version: 0.1.1
```

Comment:
why 0.1.1 and not 0.1.0 as semver suggest for initial version?
see: https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase